src/base/bittorrent/session.h

```diff
@@ -727,7 +727,7 @@ namespace BitTorrent
         QString m_resumeFolderPath;
         QFile *m_resumeFolderLock = nullptr;
 
-        QTimer *m_refreshTimer = nullptr;
+        bool m_refreshEnqueued = true;
```

Comment:
With `true` as default aren't you going to make `Q_ASSERT(!m_refreshEnqueued);` fail?

Reply:
was testing with release build sorry